health_check/cache/backends.py

```diff
@@ -5,6 +5,10 @@
 
 
 class CacheBackend(BaseHealthCheckBackend):
+
+    # Cache key used to write and read against
+    HEALTH_CHECK_KEY = "djangohealtcheck_test"
```

Comment:
Why not make it configurable via some Django setting e.g. `HEALTH_CHECK = {'CACHE_CHECK_KEY': 'my-cache-check-key'}`?

Reply:
@skarzi well, your right. My requirement was to _read_ the key's name, but changing it to another custom one would might solve other problems, too.
Also, if setting the key's name within the Django settings would still allow the developer to access it, so my requirement would still be solved :+1: 

I've updated by PR.

Reply:
I've choosen to use `HEALTHCHECK_CACHE_KEY` instead of extending `HEALTH_CHECK`.
It seems that `HEALTH_CHECK` is only used for `psutils` and other "single" settings exists (e.g. `HEALTHCHECK_CELERY_PING_TIMEOUT`.
I hope that was the right decision.